rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -8324,10 +8355,11 @@ bool opcode_02cc(const ScriptArguments& args, const ScriptObject object) {
 	@arg arg2 
 */
 void opcode_02cd(const ScriptArguments& args, const ScriptLabel arg1, const ScriptLabel arg2) {
-	RW_UNIMPLEMENTED_OPCODE(0x02cd);
-	RW_UNUSED(arg1);
 	RW_UNUSED(arg2);
-	RW_UNUSED(args);
+	/// @todo determine what arg2 is used for
```

Comment:
How about *not* making it `RW_UNUSED` then?
Warnings are good if they mean something

(Can we `-Werror` but keep these as warnings?)